global-hooks/discovery/modules_images_tags.go

```diff
@@ -30,57 +30,57 @@ var re = regexp.MustCompile(`^([0-9]+)-([a-zA-Z-]+)$`)
 
 var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 	OnStartup: &go_hook.OrderedConfig{Order: 10},
-}, discoveryModulesImagesTags)
+}, discoveryModulesImagesDigests)
 
-func discoveryModulesImagesTags(input *go_hook.HookInput) error {
+func discoveryModulesImagesDigests(input *go_hook.HookInput) error {
 	var externalModulesDir string
 
-	tagsFile := "/deckhouse/modules/images_tags.json"
+	digestsFile := "/deckhouse/modules/images_digests.json"
```

Comment:
Shouldn't we support both cases for external modules? I guess external module could be done with image tag only, then this hook will fail